rules-tests/CodeQuality/Rector/If_/ConsecutiveNullCompareReturnsToNullCoalesceQueueRector/Fixture/throw_after_if.php.inc

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Rector\Tests\CodeQuality\Rector\If_\ConsecutiveNullCompareReturnsToNullCoalesceQueueRector\Fixture;
+
+final class ThrowAfterIf
+{
+    protected ?\stdClass $prop1 = null;
+    protected ?\stdClass $prop2 = null;
+
+    public function getProp(): \stdClass
+    {
+        if ($this->prop1 !== null) {
+            return $this->prop1;
+        }
+        if ($this->prop2 !== null) {
+            return $this->prop2;
+        }
+        throw new \RuntimeException('No prop is set');
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\CodeQuality\Rector\If_\ConsecutiveNullCompareReturnsToNullCoalesceQueueRector\Fixture;
+
+final class ThrowAfterIf
+{
+    protected ?\stdClass $prop1 = null;
+    protected ?\stdClass $prop2 = null;
+
+    public function getProp(): \stdClass
+    {
+        return ($this->prop1 ?? $this->prop2) ?? new \RuntimeException('No prop is set');
```

Comment:
return throw seems got fatal error, with or without parentheses:

ref https://3v4l.org/JKLAh vs https://3v4l.org/V9iA1

so probably better to skip it.

Reply:
Oh, I see, throw Expr can actually be used

Reply:
Fixed https://github.com/rectorphp/rector-src/pull/4107/commits/8b73ec29a8f5f5684996e6c066803c7c6588918c